.github/workflows/continuous-integration.yml

```diff
@@ -33,11 +33,9 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - name: Install dependencies
         run: npm install
-      - name: Build
-        run: npm run build
       - name: Setup tests
         run: npm run pretest
       - name: Test
-        run: npm run test:coverage
+        run: npm run test
```

Comment:
If we are using `test` and not `test:coverage` then we can remove the `pretest` and `posttest` steps as `npm` runs them automatically when `npm test` is called.

Same for `.travis.yml`